.env.test

```diff
@@ -7,3 +7,4 @@ POSTGRES_PORT=54320
 POSTGRES_USER=root
 SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN=SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN
 SLACK_SIGNING_SECRET=SLACK_SIGNING_SECRET
+EMAIL_PATTERNS=@manomano.com,@prt.manomano.com
```

Comment:
Maybe, it could be better to use `EMAIL_DOMAIN_NAMES` instead of `EMAIL_PATTERNS` and remove the `@` sign from the values.

Reply:
And modify the code to take into account the removal of `@` :pray:

Reply:
Since we are not using the open source version at ManoMano (no breaking change for us), you can remove the default values and put `my-domain.com,ext.my-domain.com` as examples.

Reply:
Modifications have been done